src/ape/managers/converters.py

```diff
@@ -184,23 +169,39 @@ class ConversionManager:
     plugin_manager: ClassVar[PluginManager] = injected_before_use()  # type: ignore
     networks: ClassVar[NetworkManager] = injected_before_use()  # type: ignore
 
+    def __init__(self) -> None:
+        """
+        Handles the injected properties for ConverterAPI classes
+        """
+        for converter in (
+            AddressAPIConverter,
+            HexAddressConverter,
+            HexConverter,
+            ListTupleConverter,
+            TimestampConverter,
+        ):
+            converter.config = self.config  # type: ignore
+            converter.networks = self.networks  # type: ignore
+            converter.converter = self  # type: ignore
+
     def __repr__(self):
         return "<ConversionManager>"
 
     @cached_property
     def _converters(self) -> Dict[Type, List[ConverterAPI]]:
-        list_tuple_converter.converter = self  # Splice this in (no need for the rest)
+        ListTupleConverter.converter = self  # Splice this in (no need for the rest)
         converters: Dict[Type, List[ConverterAPI]] = {
-            AddressType: [address_api_converter, hex_address_converter],
-            bytes: [hex_converter],
-            int: [timestamp_converter],
+            AddressType: [AddressAPIConverter(), HexAddressConverter()],
+            bytes: [HexConverter()],
+            int: [TimestampConverter()],
             Decimal: [],
-            list: [list_tuple_converter],
-            tuple: [list_tuple_converter],
+            list: [ListTupleConverter()],
+            tuple: [ListTupleConverter()],
         }
 
         for plugin_name, (conversion_type, converter_class) in self.plugin_manager.converters:
-            converter = converter_class(self.config.get_config(plugin_name), self.networks, self)
+
+            converter = converter_class()
```

Comment:
For this to work, I think you have to inject into `ConversionAPI` inside of `__init__`